examples/custom_cred_provider.py

```diff
@@ -0,0 +1,27 @@
+from databricks import sql
+from databricks.sdk.oauth import OAuthClient
```

Comment:
This import only works if databricks-sdk is installed, which is not currently part of `pyproject.toml`. I can show you how to add this requirement and regenerate poetry.lock. Else we should note in a docstring or comment that users should `pip install databricks-sdk`

Reply:
I added a comment in the file and readme. Thanks for pointing it out!